fluentui_core/src/main/java/com/microsoft/fluentui/view/TemplateView.kt

```diff
@@ -38,8 +37,7 @@ import com.microsoft.fluentui.theming.FluentUIContextThemeWrapper
  */
 abstract class TemplateView : ViewGroup {
     @JvmOverloads
-    constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(FluentUIContextThemeWrapper(context), attrs, defStyleAttr)
```

Comment:
why are we not wrapping here?

Reply:
The context received in constructor is already wrapped with the theme of specific module. E.g for from tablayout we call TemplateView constructor by wrapping in Theme_FluentUI_TabLayout. If we wrap it here again without passing any theme, the default theme is applied (Base_Theme_FluentUI) to the context of say Tablayout, which is not correct